components/accelerator/nvidia/infiniband/component_output.go

```diff
@@ -103,7 +103,7 @@ var (
 // We DO NOT auto-detect infiniband devices/PCI buses, strictly rely on the user-specified config.
 func (o *Output) Evaluate(cfg infiniband.ExpectedPortStates) (string, bool, error) {
 	// nothing specified for this machine, gpud MUST skip the ib check
-	if cfg.AtLeastPorts == 0 && cfg.AtLeastRate == 0 {
+	if cfg.AtLeastPorts == 0 && cfg.AtLeastRateGb == 0 {
```

Comment:
No... Keep AtLeastRate as it is. GB is the standard Nvidia B. check out gb200 nvl72.